.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -23,6 +23,7 @@ the requirements below.
 - [ ] PR description included
 - [ ] `yarn test` passes
 - [ ] Tests are updated
+- [ ] `sideEffects` field in `package.json` is updated
```

Comment:
I'm not sure how I feel about this being here. I feel like we could address the side effects in a different way. Do we think this will catch side effects being introduced? My hunch is that it will just get ignored by people who don't understand what it means.

Reply:
I wouldn't say this will catch new introduced side effects. It is more like a reminder

Reply:
I cannot think of a different way but people will find their files with side effects are not included in the build during dev 🤣

Reply:
What if we change it to `"No side effects or `sideEffects` field updated`, with a link to docs on avoiding sideeffects in contributing? That way we are raising awareness with a more detailed explanation